hiera/hieradata/dev/wso2/wso2as/5.3.0/default.yaml

```diff
@@ -0,0 +1,94 @@
+---
+classes:
+  - wso2as
+wso2::pack_filename: "%{::product_name}-%{::product_version}.zip"
+wso2::pack_extracted_dir: "%{::product_name}-%{::product_version}"
+
+wso2::template_list :
+  - repository/conf/axis2/axis2.xml
+  - repository/conf/carbon.xml
+  - repository/conf/registry.xml
+  - repository/conf/datasources/master-datasources.xml
+  - repository/conf/tomcat/catalina-server.xml
+  - bin/wso2server.sh
+
```

Comment:
These templates are now moved to common.yaml. Please remove these templates from product specific yaml.